package.json

```diff
@@ -46,7 +52,7 @@
     "web3"
   ],
   "peerDependencies": {
-    "@prb/math": "4.0.1"
+    "@prb/math": "4.0.2"
```

Comment:
Shouldn't this be `4.x`? Or `4.0.x`?

Just thinking out loud

Reply:
not sure what you mean:

<img width="350" alt="image" src="https://github.com/sablier-labs/v2-core/assets/99738872/d499660c-1c61-4ed7-b3f4-41f9ed2cd33d">

Reply:
There's a difference between `4.0.2` and `4.x` and `4.0.x` in the context of peer dependencies.

https://chat.openai.com/share/cc9100d2-7294-42bc-b579-a3ef5c79e194

Anyway, we can leave it as `4.0.2` for now; this was more so a reminder for me to have a look later on.